htlcswitch/switch.go

```diff
@@ -737,6 +747,10 @@ func (s *Switch) handleLocalDispatch(pkt *htlcPacket) error {
 		if err != nil {
 			s.indexMtx.RUnlock()
 
+			if err := s.control.Fail(pkt.circuit.PaymentHash); err != nil {
```

Comment:
Has this payment really failed though? We're just unable to fetch the links, so an outbound HTLC w/ this payment hash hasn't even left the switch yet.